packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -202,9 +202,8 @@ export class ForkChoice implements IForkChoice {
       const blockSummary = toBlockSummary(headNode);
       if (timer) timer();
```

Comment:
Now with the finally block there's no need to also stop the timer in the try block